src/lib.rs

```diff
@@ -97,9 +109,11 @@ where
             self.write_byte(item.r);
             self.write_byte(item.b);
         }
-        // Get a timeout period of 300 ns
-        for _ in 0..900 {
-            block!(self.timer.wait()).ok();
+
+        // Latch for > 6000 ns
+        // 18 * 333 = 6327ns
+        for _ in 0..19 {
```

Comment:
It seems like there are different variations with different reset periods, see https://blog.particle.io/2017/05/11/heads-up-ws2812b-neopixels-are-about-to-change/

Reply:
Ah perfect. Thanks for the link, So then the old code was spot on there, but with a bad comment. Ive updated comments and added a link to that particle page as well.